public/src/GRPCManager.ts

```diff
@@ -78,20 +84,20 @@ export class GrpcManager {
         return this.grpcSend<Void>(this.agService.updateCourse, course)
     }
 
-    public getCourse(courseID: number): Promise<IGrpcResponse<Course>> {
-        const request = new CourseRequest()
-        request.setCourseid(courseID)
+    public getCourse(courseID: bigint): Promise<IGrpcResponse<Course>> {
```

Comment:
This comment applies to all RPC forwarding functions in `GRPCManager.ts` and `MockGRPCManager.ts`. If you instead keep the arguments to these functions as `number` and convert them to `bigint` in these functions, you can continue to use them as `number` elsewhere in the code.

I guess the drawback is that the response types still contain the `bigint` variants, which would then have to be converted back to `number` if using data from the response type as input to these functions.

I guess it is something to consider.